packages/axios-extension/src/abap/adt-catalog/services/transportcheck-service.ts

```diff
@@ -65,6 +65,91 @@ export class TransportChecksService extends AdtService {
         return this.getTransportRequestList(response.data);
     }
 
+    /**
+     * TransportChecksService API function to fetch package and transport of deployed application.
+     *
+     * @param namespace The namespace for the deployed application
+     * @returns Object that contains the package and the tansport used for the application deployment
+     */
+    public async getPackageAndTransportRequest(namespace: string): Promise<TransportCheck> {
+        const acceptHeaders = {
+            headers: {
+                Accept: 'application/vnd.sap.as+xml; dataname=com.sap.adt.transport.service.checkData',
+                'content-type':
+                    'application/vnd.sap.as+xml; charset=UTF-8; dataname=com.sap.adt.transport.service.checkData'
+            }
+        };
+
+        const data = `
+			<?xml version="1.0" encoding="UTF-8" ?>
+			<asx:abap version="1.0" xmlns:asx="http://www.sap.com/abapxml">
+				<asx:values>
+					<DATA>
+						<OPERATION>I</OPERATION>
+						<URI>/sap/bc/adt/ui_flex_dta_folder?name=${namespace}&amp;layer=CUSTOMER_BASE&amp;package=undefined</URI>
```

Comment:
Is it correct to hard code the layer to `CUSTOMER_BASE` or should that be determined by the namespace i.e. does it have the `customer.` prefix or not.

Reply:
This is no longer relevant since the method was removed.